src/lib/serviceRoutines/postContextEntitiesByEntityId.cpp

```diff
@@ -41,6 +40,10 @@
 * postContextEntitiesByEntityId - 
 *
 * POST /ngsi9/contextEntities/{entityId}
```

Comment:
My intention was to put BOTH ways to get there in the header.
Both '/v1/register' and '/ngsi9/'

We could remove the /ngsi9 and /ngsi10 completely if you like. I don't have any problem with that.